### PR TITLE
Support ReactRedux 6

### PR DESCRIPTION
**Pull request by abettadapur**
#27

### CODE REVIEW

File: packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx [hunk 1]
```diff
@@ -38,14 +43,61 @@ export class DynamicModuleLoader<
     context: IDynamicModuleLoaderContext
   ) {
     super(props, context);
-    const { modules } = props;
-    const store: IModuleStore<any> = context.store;
-    if (!store) {
-      throw new Error("Could not load store from React context.");
+    if (!ReactReduxContext) {
```
Comment (navneet-g) [inline packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx]:
```
if (!ReactReduxContext && !context.store) {
    throw new Error("Could not load store from React context.");
}
```

File: packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx [hunk 2]
```diff
@@ -38,14 +43,61 @@ export class DynamicModuleLoader<
...
+  /**
+   * Render a Redux provider
+   */
+  public render(): React.ReactNode {
+    if (ReactReduxContext) {
+      return (
+        <ReactReduxContext.Consumer>
+          {(context) => (
+            <DynamicModuleLoaderImpl
+              store={context.store}
```
Comment (navneet-g) [inline packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx]: Do we need to assert here that ```context.store``` is not null?

File: packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx [hunk 3]
```diff
@@ -6,11 +6,16 @@ import {
   IDynamicallyAddedModule
 } from "./Contracts";
 
+// @ts-ignore
```
Comment (navneet-g) [inline packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx]: why is this ```@ts-ignore```?

File: packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx (hunk 2 above)
Comment (abettadapur) [reply packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx]: Yeah, thanks

**Comment by abettadapur** [comment]
Adding a test comment

**Comment by abettadapur** [comment]
@navneet-g Can you review this once more?

File: packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx (hunk 3 above)
Comment (abettadapur) [reply packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx]: Needed, because the context is not used

File: lerna.json [hunk 4]
```diff
@@ -2,5 +2,6 @@
   "packages": [
     "packages/*"
   ],
+  "hoist": true,
```
Comment (navneet-g) [inline lerna.json]: What is this?

File: packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx [hunk 5]
```diff
@@ -40,20 +37,83 @@ export class DynamicModuleLoader<
...
+      )
+    }
+  }
+}
+
+interface IDynamicModuleLoaderImplProps {
+  /** Modules that need to be dynamically registerd */
+  modules: IModule<any>[];
+
+  store: IModuleStore<any>;
```
Comment (navneet-g) [inline packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx]:
```
store?: IModuleStore<any>;
```
To explicity state that store can be undefined.

**Pull request merged**